### ISSUE TITLE
Documentation for RHEL/Rocky

### ISSUE THREAD

**Issue by bordencastle** [SeleniumHQ/docker-selenium#1864]
### Feature and motivation

All the containers appear to be based on Debian. However, for compliance a lot of us have to build from a common OS, which is sometimes RHEL. It would be convenient to reuse most of the code even if organization forces use of different base OS.

### Usage example

Show changes needed to files in documentation to build FROM RHEL/Rocky Linux.

**Comment by diemol** [comment]
All containers are based on Ubuntu, because it offers stability (LTE) and all needed binaries for x86 are available in a simple way.

We do not have access to an environment with RHEL, so that work would need to be done by someone who has that OS available. It is a work of trial an error, I would say. Starting with changing the base docker image for one that has RHEL, and continuing from there. Seems you have access to that, would you like to help with this issue?

**Comment by bordencastle** [comment]
Yeah, I'm interested in it. I'll start testing it to see what shakes out, but not sure my results will be in real time, as I have a lot of kids and a day job.

**Comment by diemol** [comment]
Sounds good, thank you.

> Yeah, I'm interested in it. I'll start testing it to see what shakes out, but not sure my results will be in real time, as I have a lot of kids and a day job.

Absolutely, like many others in this project 😄

**Comment by bordencastle** [comment]
The base container seemed pretty straight forward, but standard RHEL UBI8 container doesn't have all the repositories that provide xvfb for example that is used in NodeBase. RHEL UBI8 is probably not the way to go, as distribution of it might need licenses to accomodate the extra repositories. Due to licensing restrictions, moving forward with this for open source purposes is probably not ideal. However, Rocky Linux may still be ideal.

**https://developers.redhat.com/articles/ubi-faq#legal_and_licensing**
41. Can I add non-UBI packages if something is missing from UBI?
Yes, but not if you want to freely redistribute the images. Once you add RHEL RPMs onto a UBI image, you are back to redistributing content released under the Red Hat Enterprise Linux end user license. If you are a paying Red Hat customer, this would break the agreement between you and Red Hat. Furthermore, receivers of these images wouldn’t receive updates for the RPMs you added unless they have Red Hat subscriptions. This puts those end users without Red Hat subscriptions in a bad place.

If you need extra packages, don’t add Red Hat Enterprise Linux packages (because they are restricted). Also, don’t add CentOS packages (because they will remove supportability). Adding CentOS packages turns the image into a Frankenstein. Neither Red Hat nor the community will want to support it. You are better off with all UBI or CentOS content. Don’t mix and match.
